### DISCUSSION TITLE
萌新求代码

### DISCUSSION THREAD

**Discussion by ghost** [VoxHwa/LM_auto_run#2] [General]
能不能求一下完整的用法 萌新不太会用 谢谢大佬

**Comment by VoxHwa** [comment]
> 能不能求一下完整的用法 萌新不太会用 谢谢大佬

😂哪里不懂，简单说就是把手动模拟定位变成全自动

 __如果以后有空__ 我尽量出个 __详细的保姆级__ 教程 
目前考虑录 __视频__ ，或者用 __GIF__ 分步解释

**Reply by ghost** [reply]
就是这个怎么用 不会运行

**Reply by ghost** [reply]
还想问一下他们那个扫码授权是怎么做到的？
